docs-src/api-params.md

```diff
@@ -0,0 +1,120 @@
+## navigation-wait-until
+- `waitUntil` <"load"|"domcontentloaded"|"networkidle"> When to consider operation succeeded, defaults to `load`. Events can be either:
+  - `'domcontentloaded'` - consider operation to be finished when the `DOMContentLoaded` event is fired.
+  - `'load'` - consider operation to be finished when the `load` event is fired.
+  - `'networkidle'` - consider operation to be finished when there are no network connections for at least `500` ms.
+
+## navigation-timeout
+- `timeout` <[number]> Maximum operation time in milliseconds, defaults to 30 seconds, pass `0` to disable timeout. The default value can be changed by using the [browserContext.setDefaultNavigationTimeout(timeout)](#browsercontextsetdefaultnavigationtimeouttimeout), [browserContext.setDefaultTimeout(timeout)](#browsercontextsetdefaulttimeouttimeout), [page.setDefaultNavigationTimeout(timeout)](#pagesetdefaultnavigationtimeouttimeout) or [page.setDefaultTimeout(timeout)](#pagesetdefaulttimeouttimeout) methods.
+
+## wait-for-timeout
+- `timeout` <[number]> maximum time to wait for in milliseconds. Defaults to `30000` (30 seconds). Pass `0` to disable timeout. The default value can be changed by using the [browserContext.setDefaultTimeout(timeout)](#browsercontextsetdefaulttimeouttimeout).
```

Comment:
Hmm... Why doesn't this mention page.setDefaultTimeout? Perhaps this is different for page.waitForX and context.waitForX?

Reply:
That's what it was!